lib/ansible/modules/windows/win_updates.ps1

```diff
@@ -60,6 +61,7 @@ $update_script_block = {
     Function Start-Updates {
         Param(
             $category_guids,
+            [string[]]$post_categories=@(),
```

Comment:
Do you need to add `[AllowEmptyCollection()]` for this?

Reply:
As long as there's an empty list being provided then all should be fine.